[2025-02-02T16:05:53.446+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T16:05:53.470+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T16:05:50.446357+00:00 [queued]>
[2025-02-02T16:05:53.479+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T16:05:50.446357+00:00 [queued]>
[2025-02-02T16:05:53.482+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-02-02T16:05:53.496+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): get_data_fly> on 2025-02-02 16:05:50.446357+00:00
[2025-02-02T16:05:53.504+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=258) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T16:05:53.505+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'get_data_fly', 'manual__2025-02-02T16:05:50.446357+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmp_t383jvq']
[2025-02-02T16:05:53.506+0000] {standard_task_runner.py:72} INFO - Started process 290 to run task
[2025-02-02T16:05:53.507+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask get_data_fly
[2025-02-02T16:05:53.549+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.get_data_fly manual__2025-02-02T16:05:50.446357+00:00 [running]> on host d628bd12729b
[2025-02-02T16:05:53.623+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='get_data_fly' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T16:05:50.446357+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T16:05:50.446357+00:00'
[2025-02-02T16:05:53.625+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T16:05:55.592+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T16:05:55.613+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T16:05:55.615+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=get_data_fly, run_id=manual__2025-02-02T16:05:50.446357+00:00, execution_date=20250202T160550, start_date=20250202T160553, end_date=20250202T160555
[2025-02-02T16:05:55.858+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T16:05:55.918+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T16:05:55.927+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
